lib/modules/platform/bitbucket/index.ts

```diff
@@ -952,11 +955,58 @@ export async function createPr({
         renovateUserUuid,
         pr,
       );
+      if (platformPrOptions?.bbAutoResolvePrTasks) {
+        await autoResolvePrTasks(pr);
+      }
       return pr;
     }
   }
 }
 
+async function autoResolvePrTasks(pr: Pr): Promise<void> {
+  logger.debug(`Auto resolve PR tasks in #${pr.number}`);
+  try {
+    const unResolvedTasks = (
+      await bitbucketHttp.getJson(
+        `/2.0/repositories/${config.repository}/pullrequests/${pr.number}/tasks`,
+        { paginate: true, pagelen: 100 },
+        UnresolvedPrTasks,
+      )
+    ).body;
+
+    logger.trace(
+      {
+        prId: pr.number,
+        listTaskRes: unResolvedTasks,
+      },
+      'List PR tasks',
+    );
+
+    for (const task of unResolvedTasks) {
+      const res = await bitbucketHttp.putJson(
+        `/2.0/repositories/${config.repository}/pullrequests/${pr.number}/tasks/${task.id}`,
+        {
+          body: {
+            state: 'RESOLVED',
+            content: {
+              raw: task.content.raw,
+            },
```

Comment:
do you really need to send the contents again? 🤔

Reply:
Yep, it's marked as required https://developer.atlassian.com/cloud/bitbucket/rest/api-group-pullrequests/#api-repositories-workspace-repo-slug-pullrequests-pull-request-id-tasks-task-id-put

<img width="1218" alt="image" src="https://github.com/user-attachments/assets/f78737fa-4f93-4683-97ed-a45e1c7bcc03">